src/ng-workspace.ts

```diff
@@ -59,6 +60,7 @@ export interface NgPublishToGitConfig {
 
 export interface PackageInfo {
   version: string;
+  prePublishToGit: string;
```

Comment:
It would be better if the 'prePublishToGit' key is below the scripts key, in that case it can also be executed by 'npm run prePublishToGit' from a command prompt to test your script, it then behaves just like any npm script.